src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
```

Comment:
You have attached an event listener to the '.logo' element without checking if the element exists. This could cause an error if the element is not found. Always check if the element exists before attaching an event listener.